src/openseadragon.js

```diff
@@ -1354,6 +1354,8 @@ function OpenSeadragon( options ){
             maxImageCacheCount:     200,
             timeout:                30000,
             useCanvas:              true,  // Use canvas element for drawing if available
+            tileRetryMax:           0,
```

Comment:
I'm realizing these need to be documented. Somewhere around here would be good: 

https://github.com/openseadragon/openseadragon/blob/d7cd12679c77c96a5e114f353862618ea9c55ad9/src/openseadragon.js#L497

Reply:
Will do